src/distributing-policies/secure-supply-chain.md

```diff
@@ -0,0 +1,387 @@
+# Secure Supply Chain
+
+Users can configure Kubewarden to run policies signed by only entities that they
```

Comment:
I would rephrase this paragraph as: (note the different usage of `only`)

> Cluster Operators can configure Kubewarden to only run policies signed by entities that they
trust, to ensure the validity of the policies' chain of custody.  
Policy developers can sign their policies and publish them in a registry.
Signing and verifying policies and container images is achieved via the [Sigstore](https://www.sigstore.dev) project.

And maybe add a short description of what a Secure Supply Chain is or entails.